cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -165,6 +166,26 @@ def direct_relation_must_be_nullable(cls, value: Any, info: ValidationInfo) -> N
             raise ValueError("Direct relation must be nullable")
         return value
 
+    @field_validator("container_property", "container")
+    def reverse_direct_relation_has_no_container(cls, value, info: ValidationInfo) -> None:
+        if info.data.get("relation") == "reversedirect" and value is not None:
+            raise ValueError(f"Reverse direct relation must not have container or container property, got {value}")
+        return value
+
+    @field_validator("value_type", mode="after")
+    def relations_value_type(cls, value: CdfValueType, info: ValidationInfo) -> CdfValueType:
+        if (relation := info.data["relation"]) is None:
+            return value
+        if not isinstance(value, ViewPropEntity):
+            raise ValueError(f"Relations must have a value type that points to another view, got {value}")
+        if relation == "reversedirect" and value.property_ is None:
```

Comment:
I see you are storing here information about reverse property, which should be called `symmetric property`

Reply:
or simply call it reversDirect or reverseProperty